test/stancjs/stancjs.expected

```diff
@@ -68,17 +68,11 @@ model {
   y ~ normal(((((((((0)))))))), 1);
 }
 
-parameters {
-  real y;
-}
-model {
-  if (y) {
-    print("y is nonzero ");
-  }
-  
-  y ~ normal(0, 1);
-}
+Error in 'string', line 6, column 8: Condition of type real is deprecated and
+    will be disallowed in Stan 2.34. Use an explicit != 0 comparison instead.
+    This can be automatically changed using the canonicalize flag for stanc
 
+undefined
```

Comment:
Stancjs tests are breaking.

Reply:
I left that in on purpose since testing the deprecations part of the canonicalizer is still nice, and we have basically no deprecated features we can canonicalize right now that _aren't_ removed. The equivalent of this PR in 2.35 will need to update it, of course